postgres/README.md

```diff
@@ -17,23 +18,82 @@ The PostgreSQL check is packaged with the Agent. To start gathering your Postgre
 
 ### Configuration
 
-#### Prepare Postgres
+#### Postgres parameters
 
-To get started with the PostgreSQL integration, create a read-only `datadog` user with proper access to your PostgreSQL server. Start `psql` on your PostgreSQL database.
+Configure the following [Postgres parameters](https://www.postgresql.org/docs/13/config-setting.html):
 
-For PostgreSQL version 10 and above, run:
+```ini
+# Add pg_stats_statements to shared_preload_libraries for collection of query statistics
+shared_preload_libraries = pg_stats_statements
 
-```shell
-create user datadog with password '<PASSWORD>';
-grant pg_monitor to datadog;
-grant SELECT ON pg_stat_database to datadog;
+# Collect larger query text in pg_stat_activity and pg_stat_statements
+track_activity_query_size = 4096
+
+# Collect a wider variety of queries in pg_stat_statements
+pg_stat_statements.max = 10000
+pg_stat_statements.track = ALL
 ```
 
-For older PostgreSQL versions, run:
+* For self-hosted instances these parameters are configured in `postgresql.conf`
+* For AWS [RDS](https://aws.amazon.com/rds/postgresql/) or [RDS Aurora](https://aws.amazon.com/rds/aurora/postgresql-features/) these parameters are configured through the [DB Parameter Group](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_WorkingWithParamGroups.html)
 
-```shell
-create user datadog with password '<PASSWORD>';
-grant SELECT ON pg_stat_database to datadog;
+#### Datadog user setup
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Postgres ≤ 9.6" xxx -->
+
+```SQL
+CREATE USER datadog WITH password '<PASSWORD>';
+CREATE SCHEMA datadog;
+GRANT USAGE ON SCHEMA datadog TO datadog;
+GRANT USAGE ON SCHEMA public TO datadog;
+GRANT SELECT ON pg_stat_database TO datadog;
+```
```

Comment:
This section doesn't have any context. It just drops some SQL in front of you. It's also important to clarify what the required permissions are of the user who is going to be executing the SQL.